uber/templates/jobs/main_menu.html

```diff
@@ -14,7 +14,7 @@
     </td>
     <td width="50%">
         <select id="page" style="font-size:14pt ; font-weight:bold" onChange="mainMenuDropdownChanged()">
```

Comment:
Ahh, I see what happened.  This is a ``c`` issue.  ``c.PAGE_PATH`` is the full path to the page starting from the mount point, e.g. ``/jobs/signups`` but ``c.PAGE`` is just the last part, e.g. ``signups``.  So you don't need to return this from the page handler to get it into the template, since it's already there.